scripts/HNG-02083-startoffs.js

```diff
@@ -0,0 +1,6 @@
+let fullName = "Morounfolu Adesanya";
+let HNGi7_ID = "HNG-02083";
+let language = "Javascript";
+let email = "adelekesays@gmail.com";
+const MESSAGE = `Hello World,this is ${fullName} with HNGi7 ID ${HNGi7_ID} and email ${email} using ${language} for stage 2 task.`;
```

Comment:
Please move the email to the end of the string after the word `task` as highlighted [here](https://hngi7.slack.com/archives/C014G2TMQ3G/p1591185461011700)

Reply:
Okay, amendment has been done.